src/js/components/TaskViewComponent.jsx

```diff
@@ -45,7 +45,7 @@ var TaskViewComponent = React.createClass({
     AppsActions.requestApp(this.props.appId);
   },
 
-  handleKillSelectedTasks: function (scaleTask) {
+  handleKillSelectedTasks: function (scaleTask = false, wipeTasks = false) {
```

Comment:
I realised `wipeTasks` is implicitly being sent the `SyntheticEvent` by the regular `Kill` and `Kill & Scale` buttons. Will change this.